Makefile

```diff
@@ -0,0 +1,153 @@
+# Makefile to help automate tasks
+PY := bin/python
+PIP := bin/pip
+PYTEST := bin/py.test
+THEBLUES := lib/python*/site-packages/theblues.egg-link
+FLAKE8 := bin/flake8
+SPHINX := bin/sphinx-apidoc
```

Comment:
Originally it had a marker showing the file did not end with a  new line.  Sorry for the confusing comment.